build/installers/liferay-project-sdk-with-devstudio-ce/liferay-project-sdk-with-devstudio-ce.xml

```diff
@@ -68,10 +68,230 @@
         <include file="../components/autodetect-java.xml" />
     </preInstallationActionList>
     <postInstallationActionList>
-        <createDirectory>
-            <path>${user_home_directory}${platform_path_separator}.liferay</path>
-            <progressText>Creating directory...</progressText>
-        </createDirectory>
+        <if>
+            <actionList>
+                <createDirectory>
+                    <path>${user_home_directory}${platform_path_separator}.liferay</path>
+                    <progressText>Creating directory...</progressText>
+                </createDirectory>
+                <setInstallerVariable>
+                    <name>liferayBundlesPath</name>
+                    <value>${user_home_directory}${platform_path_separator}.liferay${platform_path_separator}bundles</value>
+                </setInstallerVariable>
+                <createDirectory>
+                    <path>${liferayBundlesPath}</path>
+                    <progressText>Creating directory...</progressText>
+                </createDirectory>
+                <copyFile>
+                    <destination>${liferayBundlesPath}${platform_path_separator}liferay-ce-portal-tomcat-7.1-a1-20180410105031513.zip</destination>
+                    <origin>${installdir}${platform_path_separator}liferay-ce-portal-tomcat-7.1-a1-20180410105031513.zip</origin>
+                </copyFile>
+                <copyFile>
+                    <destination>${liferayBundlesPath}${platform_path_separator}com.liferay.portal.plugins.sdk-1.0.16-withdependencies.zip</destination>
```

Comment:
Need to update the plugins.sdk with dependencies to latest version.

Reply:
hey Greg
the 1.0.16 is the latest one in https://repository.liferay.com/nexus/content/repositories/liferay-public-releases/com/liferay/portal/com.liferay.portal.plugins.sdk/